resources-service/package.json

```diff
@@ -12,7 +12,7 @@
     "jest-each": "^29.2.1"
   },
   "scripts": {
-    "migrate": "run-s db:create:schema test:service:ci:migrate",
+    "migrate": "cross-env RDS_USERNAME=hrm RDS_PASSWORD=postgres RESOURCES_PASSWORD=resources-password run-s db:create:schema test:service:ci:migrate",
```

Comment:
I'm not sure this env vars are needed at all :thinking:

Reply:
@GPaoloni 
I don't think `RDS_USERNAME` is needed because it has a default set in the scripts, but the other two have defaults of `null` so they have to be set for that migrate function to run properly in my local dev env.

Was I missing something else?

Reply:
Wait... is there meant to be a .env file for resources? Should that be added to `ssm:local`?

Reply:
I believe they will default to some stuff too, as described in `resources-service/src/config/db.ts`:
```
export default {
  username: process.env.RESOURCES_USERNAME || 'resources',
  password: process.env.RESOURCES_PASSWORD || '',
  database: process.env.RDS_DBNAME || 'hrmdb',
  ...
}
```

I may be wrong tho, and in any case those env vars don't affect.

> Wait... is there meant to be a .env file for resources? Should that be added to ssm:local?

I am not sure but I think no?

Reply:
I am very confused. I just removed it and everything still works,  but I'm not sure HOW it is still working since `resources-service/create-db.js` and `resources-service/db-migrate.js` don't appear to use db.ts?

Reply:
/shrug I'm merging without the env files since it is working for me that way. Will revisit later if i find the error I was encountering again.